### ISSUE TITLE
Feature: deployment setup for `scorecard-site`

### ISSUE THREAD

**Issue by azeemshaikh38** [ossf/scorecard-webapp#58]
I plan to make some changes to how `scorecard-site` gets deployed. This is what I want to do:

1. Have 2 AppEngine configurations: `prod` and `staging`.
2. `main` branch in this repo will update the `staging` site while `production-site` branch will update `prod`. This will be done through CloudBuild triggers.
3. Branch protection setting for both `main` and `production-site` will require code review and approvals. Any changes which need to be pushed to production site, will require a PR from `main` to `production-site`.

We can carry over this process when we are ready to port over to Netlify.

@mdunbavan @ossf/scorecard-maintainers FYI.

**Comment by mdunbavan** [comment]
@azeemshaikh38 this all seems legit and a good way going forward for now.

**Comment by spencerschrock** [comment]
> @spencerschrock is this info up to date?

No. There is just the Netlify deploy and all commits to `main` get deployed to the site. I suppose the deploy preview on PRs is the "staging"
